build.sh

```diff
@@ -37,17 +37,14 @@ packages() {
 	cd /mnt/raspotify
 
 	if [ ! -d librespot ]; then
-		# Use a vendored version of librespot.
-		# https://github.com/librespot-org/librespot does not regularly or
-		# really ever update their dependencies on released versions.
-		# https://github.com/librespot-org/librespot/pull/1068
-		echo "Get https://github.com/JasonLG1979/librespot/tree/raspotify..."
-		git clone https://github.com/JasonLG1979/librespot
-		cd librespot
-		git checkout raspotify
+		echo "Get https://github.com/librespot-org/librespot/tree/dev..."
+		git clone --branch dev https://github.com/librespot-org/librespot
 		cd /mnt/raspotify
 	fi
 
+	# Quelch warning about dubious ownership
+	git config --global --add safe.directory /mnt/raspotify/librespot
```

Comment:
Are you using Docker in rootless mode? I designed Raspotify _before_ rootless was available, and I'm noticing "dubious" ownership as well. Perhaps below we can check if the UID is high, and only apply the "permfix" if necessary. (Lines 189-190 after your changes)

```bash
# Perm fixup. Not needed on macOS, but is on Linux
chown -R "$PERMFIX_UID:$PERMFIX_GID" /mnt/raspotify 2>/dev/null || true
```